src/js/player.js

```diff
@@ -2892,47 +2941,50 @@ Player.prototype.options_ = {
   notSupportedMessage: 'No compatible source was found for this media.'
 };
 
+// The following pointless assignment expressions are here only for
+// purposes of documentation tooling compatibility.
+
 /**
  * Fired when the user agent begins looking for media data
  *
  * @event loadstart
  */
-Player.prototype.handleTechLoadStart_;
+Player.prototype.handleTechLoadStart_ = Player.prototype.handleTechLoadStart_;
```

Comment:
Does the linter complain about these? I think it would be nice to keep them as stubs, if possible.